protos/deployment.proto

```diff
@@ -181,9 +182,7 @@ message ListDeploymentsRequest {
   DeploymentSpec.DeploymentOperator operator = 4;
   SORTABLE_COLUMN order_by = 5;
   bool ascending_order = 6;
-  // Optional, currently only handle label filter with key=value pairs. In the future,
-  // we should handle more advanced filter query such as `in` and `notin`.
-  string labels_query =7;
+  LabelSelectors label_selectors =7;
```

Comment:
similar to the other gcs PR. Let's not do this in the future. It's ok for now since we are doing a larger release and notify users to update yatai service

Reply:
change to the following?
```
  string labels_query =7;  // deprecated
  LabelSelectors label_selectors =8;
```